go.mod

```diff
@@ -149,24 +215,34 @@ require (
 	google.golang.org/grpc v1.13.0
 	gopkg.in/AlecAivazis/survey.v1 v1.6.2
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
+	gopkg.in/asn1-ber.v1 v1.0.0-20170511165959-379148ca0225 // indirect
+	gopkg.in/bufio.v1 v1.0.0-20140618132640-567b2bfa514e // indirect
+	gopkg.in/editorconfig/editorconfig-core-go.v1 v1.3.0 // indirect
 	gopkg.in/fsnotify.v1 v1.4.7 // indirect
+	gopkg.in/gomail.v2 v2.0.0-20160411212932-81ebce5c23df // indirect
 	gopkg.in/inf.v0 v0.9.1
+	gopkg.in/ini.v1 v1.39.0 // indirect
+	gopkg.in/ldap.v2 v2.5.1 // indirect
+	gopkg.in/macaron.v1 v1.3.1 // indirect
+	gopkg.in/redis.v2 v2.3.2 // indirect
 	gopkg.in/robfig/cron.v2 v2.0.0-20150107220207-be2e0b0deed5
 	gopkg.in/src-d/go-billy.v4 v4.2.0
 	gopkg.in/src-d/go-git.v4 v4.5.0
+	gopkg.in/testfixtures.v2 v2.5.0 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 	gopkg.in/warnings.v0 v0.1.2
 	gopkg.in/yaml.v2 v2.2.1
 	k8s.io/api v0.0.0-20180628040859-072894a440bd
 	k8s.io/apiextensions-apiserver v0.0.0-20180621085152-bbc52469f98b
 	k8s.io/apimachinery v0.0.0-20180621070125-103fd098999d
 	k8s.io/client-go v8.0.0+incompatible
-	k8s.io/code-generator v0.0.0-20180904193909-8c97d6ab64da
+	k8s.io/code-generator v0.0.0-20181017053441-40b92a94e6ad
```

Comment:
Please could you revert this to old version?

Reply:
@ccojocar is it sufficient to just delete go.sum and go.mod in the github diff view?

Reply:
I would revert them, rebase from `master` and run a `make test`.